### PR TITLE
chore: improve SCIM documentation

### PR DESCRIPTION
**Pull request by mefellows**
- Moves authentication documentation together
- Adds redirect plugin so that existing links will continue to work
- Clarifies the conditions for the SCIM adapter to be successful, and adds a troubleshooting item

Screenshot 2023-07-26 at 5 46 02 pm

### CODE REVIEW

**Comment by mefellows** [comment]
I suppose the constraint for migrating users only applies to SaaS PactFlow. I'll reword that bit.

File: website/docs/docs/scim/main.md [hunk 1]
```diff
@@ -4,10 +4,20 @@ title: API
...
-**NOTE: SAML is a pre-requisite for this feature to work.**
-
 The SCIM 2.0 compatible API manages PactFlow users and teams. For more details on SCIM, refer to https://www.simplecloud.info/.
 
+## Pre-requisites
+
+You must have SAML configured for your PactFlow instance before users can login. See the [SAML documentation](/docs/authentication/main#saml-support) for setup instructions.
+
+:::caution PactFlow SAAS Customers
+It is recommended to only allow a small number of test users access to your SAML authentication until you have completed the SCIM integration, so that users provisioned via the SCIM connection are connected to the federated identity. If you allow all users to login via SAML prior to setting up SCIM, then users provisioned via SCIM will end up as [duplicates](#duplicate-users).
```
Comment (bethesque) [inline website/docs/docs/scim/main.md:14]: I would turn this around.

> Users already created via SAML will not be automatically linked to users created via the SCIM integration, even if the email addresses and IDP user identifiers match. For this reason, if you intend to use SCIM it is recommend to only allow a small number of test users access to your SAML authentication until you have completed the SCIM integration, to reduce the number of users who will need to be cleaned up or migrated afterwards.

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -111,12 +111,25 @@ module.exports = {
...
         "docker",
         "groovy",
-        "scala"
+        "scala",
       ],
     },
   },
   themes: ["@you54f/theme-github-codeblock"], // switch to @saucelabs/theme-github-codeblock when merged https://github.com/saucelabs/docusaurus-theme-github-codeblock/pull/19
-  plugins: [path.resolve(__dirname, "src/plugins/plugin-segment")],
+  plugins: [
```
Comment (bethesque) [inline website/docusaurus.config.js:119]: Nice! cc: @Inksprout

**Review by bethesque — APPROVED** [review]

**Comment by bethesque** [comment]
I think we have this problem in onprem too. There is no logic in the "create user" endpoint that does an upsert if the user already exists for a given identifier.

**Comment by mefellows** [comment]
Thanks Beth. I've accepted your suggestion and will release shortly.

**Pull request merged**